src/python/era_5g_object_detection_common/era_5g_object_detection_common/face_detector.py

```diff
@@ -71,3 +71,25 @@ def process_image(self, frame):
             detections_raw.append(det)
 
         return detections_raw
+
+    def process_images(self, frames):
```

Comment:
Maybe add type annotations? Not mandatory in this repo so far, but it is quite useful - certainly more useful than the description in docstring :-)